src/app/api/bridge/user/new/get-status/route.ts

```diff
@@ -9,20 +9,23 @@ export async function POST(request: NextRequest) {
             throw new Error('BRIDGE_API_KEY is not defined')
         }
 
-        const getUserFromCookieRequest = new NextRequest('/api/peanut/user/get-user-from-cookie', {
-            method: 'GET',
-            headers: {
-                cookie: request.headers.get('cookie') ?? '',
-                ...request.headers,
-            },
-        })
+        const getUserFromCookieRequest = new NextRequest(
+            `${process.env.NEXT_PUBLIC_BASE_URL}/api/peanut/user/get-user-from-cookie`,
```

Comment:
Q: Why is this code part of this PR?

Reply:
Ideally we would've pulled `peanut-main` into `peanut-wallet` , and then that into the feature branch. This would avoid bloating the PR with unrelated code additions